web/src/components/StarButton.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div class="star-container">
```

Comment:
Instead of defining a custom css class, it is better practice to use Vuetify's built-in classes
```suggestion
  <div class="d-flex align-center justify-end ml-auto">
```